pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/extension/Format.kt

```diff
@@ -28,6 +29,15 @@ fun Format.hasSelection(selection: @SelectionFlags Int): Boolean {
     return selectionFlags and selection != 0
 }
 
+/**
+ * Is forced
+ *
+ * @return true if [Format.selectionFlags] contains [C.SELECTION_FLAG_FORCED].
```

Comment:
Not sure that describing how the method is implemented adds meaningful information here.